api/src/tests/api.test.ts

```diff
@@ -164,7 +164,8 @@ describe("API", () => {
       } as Entity);
     });
 
-    test("symbol", async () => {
+    // Skipped by @codeviking on 12/10/2020.
```

Comment:
Would be nice to have a one-liner as to why :)

Reply:
Yea, I agree --  I myself don't know why.

@andrewhead Do you have details I could add here? I could also add:

```
// Skipped by @codeviking on 12/10/2002 -- appears to have been a long-standing regression, so I'm not sure why
// it stopped working.
```

Reply:
I added a comment noting explaining why it was skipped (to the best of my ability).